docs/architecture/bpmn-parsing.adoc

```diff
@@ -0,0 +1,14 @@
+= BPMN Parsing
+
+== XML Parser
+
+To parse a XML data, we use the library https://github.com/NaturalIntelligence/fast-xml-parser[fast-xml-parser].
+
+You can see the implementation in https://github.com/bonitasoft-labs/bpmn-visu-js/tree/master/src/component/parser/xml/BpmnXmlParser.ts[BpmnXmlParser.ts]
```

Comment:
❓ I am not sure we want to put a direct link to the code here (and in the rest of this page) as we don't do this in other docs. In addition, this always point to the latest development and will make old doc wrong on class move or rename
If so, we may have to update other docs for consistency.

Reply:
I'm not sure to understand what you want to say when you talk about old doc.
The doc of a branch always point the content of this branch, it's always up to date.

I am not closed to not putting code reference in the doc. But I thought that in the archi doc, we would talk about it a bit so that a new developer understands how our code is organized.

Reply:
The link here points to the master branch so is not tight to the current commit of the documentation.
It is ok at the time of writing, but if afterwards, the class moves, the 0.1.2 documentation will point to something that do not exist anymore.
I am saying we do not have to put code reference; my concern is about non permanent link and maintenance impacts of such links.

Let's talk about this in person, to conclude if you are ok.

Reply:
Decisions taken: remove link and keep class names. When the lib will be more mature, we may add links